culture/code-of-conduct.md

```diff
@@ -0,0 +1,124 @@
+* **The consequences for the 2i2c community of not reporting bad behavior outweigh the consequences for one person of reporting it.** We sometimes hear “I don’t want X person to meet consequences because I told someone about their bad behavior.” Consider the impact on everyone else at 2i2c of letting their behavior continue unchecked.
+* 2i2c only works as an open, participatory, community-driven community because of shared trust between community members. **Reporting code of conduct violations helps us identify when this trust is broken, to prevent that from happening in the future.**
+
+
+### Where and how to report
+
+Please report all code of conduct violations using [our reporting form](https://docs.google.com/forms/d/e/1FAIpQLSeKFuoghGzftqqgaa0xllZyocdUJsWZmoSaFFlerimg4n_Y8A/viewform?usp=sf_link). This is a short Google Form that will be sent to the Executive Director.
+
+If you would rather discuss the matter in person with a team member, send a direct message in the 2i2c Slack, or email a Steering Council member to schedule a time to talk.
+
+
+### Confidentiality
+
+We will keep all reports confidential, except if we've discussed with you and agreed otherwise. When we discuss incidents with people who are reported, we will anonymize details as much as we can to protect reporter privacy.
+
+However, some incidents happen in one-on-one interactions, and even if the details are anonymized, the reported person may be able to guess who made the report. If you have concerns about retaliation or your personal safety, and do not want us to share the details of your report with anyone (including the perpetrator) please let us know explicitly in your report. Unfortunately, in that situation we will not be able to take any action.
+
+In some cases we may decide to share an update about a major incident with 2i2c team members, or with the entire 2i2c community. If that's the case, the identities of all victims and reporters will remain confidential unless those individuals instruct us otherwise.
+
+
+## Social rules
+
+In addition to having a code of conduct, we have four lightweight [social rules](social-rules.md). The social rules are different and separate from the code of conduct. They help us create a better learning environment by giving names to counterproductive behavior and acting as a release valve so that frustration doesn't build up over time. We expect people to unintentionally break the social rules from time to time. Doing this doesn't make you a bad person or a bad community member. When this happens, it's not a big deal. Just apologize and move on.
+
+The enforcement provisions in this code of conduct do not apply to the social rules. We definitely won't give you a strong warning or expel you from the 2i2c community just for breaking a social rule.
+
+If you have any questions about any part of the code of conduct or social rules, please reach out to any 2i2c team member.
```

Comment:
is there a url here we could add for 2i2c/people ? showing who those people are help make it a bit more human

Reply:
good point, I'll link to the `About Us` page for now